src/components/material-description/MaterialDescription.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { useGetManifestationQuery } from "../../core/dbc-gateway/generated/graphql";
+import { generateMapId } from "../../core/utils/helpers";
+import { useText } from "../../core/utils/text";
+import { Pid } from "../../core/utils/types/ids";
+
+export interface MaterialDescriptionProps {
+  pid: Pid;
+}
+
+const MaterialDescription: React.FC<MaterialDescriptionProps> = ({ pid }) => {
+  const { data, isLoading } = useGetManifestationQuery({ pid });
+  const t = useText();
+  const baseUrl = t("baseUrlText");
+  const identifierText = t("identifierText");
```

Comment:
Please to not declare this texts separately.

If these texts are not used multiple places I think it is fine just to inline them in the JSX. The name of the `t()` function has been chosen because it is sufficiently short.

Reply:
👍 💯